Example/Example/Model/Model.swift

```diff
@@ -1,7 +1,7 @@
 import Faro
 
 /// Example model
-class Model: Parseable {
+class Model: Deserializable {
```

Comment:
Ik ga de rename doen in #81 want anders te veel conflicten. @fousa wat vind je van het voorstel?